js/background.js

```diff
@@ -153,7 +173,12 @@ chrome.runtime.onConnectExternal.addListener(function (port) {
 
     port.onMessage.addListener(function (msg) {
         if (msg.type === 'Ping') {
-            port.postMessage({type: 'Pong', enabled: isEnabled()})
+            port.postMessage({ type: 'Pong', enabled: isEnabled() })
+        } else if (msg.type === 'Url') {
+            chrome.tabs.create({
+                url: msg.value,
+                active: false,
```

Comment:
I think I'd like it if the newly opened tab was active/focused immediately. If you do `active: true` here will it focus the new tab?